dev-python/pyside/pyside-5.11.1.ebuild

```diff
@@ -0,0 +1,155 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit cmake-utils python-r1 virtualx
+
+DESCRIPTION="Python bindings for the Qt framework"
+HOMEPAGE="https://wiki.qt.io/PySide2"
+SRC_URI="https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${PV}-src/pyside-setup-everywhere-src-${PV}.tar.xz"
+
+# See "sources/pyside2/PySide2/licensecomment.txt" for licensing details.
+LICENSE="|| ( GPL-2 GPL-3+ LGPL-3 )"
+SLOT="2"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="3d charts concurrent datavis declarative designer gui help location multimedia
+	network opengl positioning printsupport script scripttools scxml sensors speech
+	sql svg test testlib webchannel webengine websockets widgets x11extras xmlpatterns"
+
```

Comment:
Nice to see, you have added USE flags for the missing modules from live ebuild. But don't you need to add them in REQUIRED_USE as well?

Reply:
yes please